Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM clearlinux:base AS build
 
-ARG VERSION
-ARG NDCTL_VERSION
-ARG NDCTL_CONFIGFLAGS
-ARG NDCTL_BUILD_DEPS
+ARG VERSION="unknown"
+ARG NDCTL_VERSION="64.1"
+ARG NDCTL_CONFIGFLAGS="--libdir=/usr/lib --disable-docs --without-systemd --without-bash"
+ARG NDCTL_BUILD_DEPS="os-core-dev devpkg-util-linux devpkg-kmod devpkg-json-c"
```

Comment:
Can you also add `--fail` to the curl invocation?

The lack of that was hiding the root cause of the build failure.

Reply:
Done